src/s1a_portapotty.tw2

```diff
@@ -80,13 +80,13 @@ I panicked, because I still couldn’t tell where it was. And now, like some Edg
 ::Jail Cell (Lit) - name character
 I stared into the mirror and nearly had a heart attack.
 
-I saw a face I didn't recognize staring back at me:
+[(live: 1s)[(if: time > 2s)[$dissolve[I saw a face I didn't recognize staring back at me:(stop:)]]]]
```

Comment:
This sequence read weird to me when it displayed all at once, so I added some time delay to each line. I like it this way, but I'm open to suggestions too.